package/Dockerfile.agent

```diff
@@ -1,7 +1,7 @@
 ARG BUILD_ENV=dapper
 
-FROM registry.suse.com/bci/bci-base:15.5.36.5.52 AS base
-RUN zypper in --no-recommends -y git bash openssh && groupadd -g 1000 fleet-apply && useradd -u 1000 -g 1000 -m fleet-apply; rm -fr /var/cache/* /var/log/*log
+FROM registry.suse.com/suse/git:2.35 AS base
```

Comment:
Dropping to a major.minor version for the tag could be done everywhere. When rebuilt (e.g., for the final) you would have the fixed packages automatically be there.

Reply:
I think automation will set a specific version when it runs 🤷